mesheryctl/internal/cli/root/system/start.go

```diff
@@ -96,159 +96,159 @@ func start() error {
 	if err != nil {
 		return errors.Wrap(err, "error processing config")
 	}
-	if(mctlCfg.Contexts["local"].Platform== "docker") 
-	{
+	ctxPlatform := mctlCfg.GetContextContent().Platform
```

Comment:
These is an addon that can be done here. from #2256 I am planning to add a context flag. Taking that into consideration for which context to focus on would be great. Another issue could be created for this.